ChangeLog

```diff
@@ -29,6 +29,33 @@ Version 8.3.0DEV
  - Add direct button to public page for registering when enabled.
  - Scale batch size of judge tasks dynamically.
  - Rename room to location for teams.
+ - Add option to upload contest problemset document.
+ - Improve on the advice for rejuding on config changes, only when relevant
+   advice to start a rejudging, similar for updating the scoreboard cache.
+ - For interactive problems: add explicit statement to when one of sides of
+   the interactions ran closed their output.
+ - Allow importing a (generic) contest warning which is displayed on the scoreboard.
+ - Added option in the dj_setup_database to make dumps, useful before major
+   upgrades and during the contest before problem imports.
+ - Fixed some bugs to become OpenAPI spec compliant again.
+ - Prevent a race condition leading to two submissions receiving the same
+   colour balloon.
+ - Compile executables with `-std=gnu++20` now.
+ - Record the testcase directory for judging runs, this makes investigating
+   easier.
+ - Set default C/C++ standards in configure script, we intend to ship all
+   scripts free of warnings/errors.
+ - Add option to not show any rank on mirror contests, this is useful when
+   you want to disclose the problems but not make comparing easy between the
+   official contest and the mirror.
+ - Allow basic authentication for the import-contest URL.
```

Comment:
This was just fixing that if you had a netrc the script still works with username + password in the URL right? In that case this wording is not very clear.

Reply:
Yes, added a commit to hopefully make this more clear.